### DISCUSSION TITLE
Can we edit our comments?

### DISCUSSION THREAD

**Discussion by nepomnyi** [c-ding-math/functor-network#21] [Q&A] [answered]
Hello @c-ding-math , I was going to edit my comment under someone else's post and couldn't find the "edit" button. Then I went ahead and checked if my own post has the "edit" button - and, indeed, it does. So, we can't edit our comments, but we can edit out posts. I am wondering if you intentionally disabled editing of the comments. If it was not intentional, I am wondering if you agree to enable editing of the comments, please.
Thank you in advance.

**Comment by c-ding-math** [comment, marked as answer]
It is intentional. 
If someone replies to your comment and you change your comment content, then the reply may look strange. So comment cannot be edited but can be deleted, and when the comment is deleted, all the replies will also be deleted.
By the way, who can delete your comment? Yourself and the post author.

**Comment by nepomnyi** [comment]
I see, thank you.

Ivan
